src/app/asset-filters/asset-filters.component.pug

```diff
@@ -41,7 +41,7 @@
   .filter-group(*ngIf="availableFilters.dateObj")
     span.filter-group__label
       h3 Date
-        a#buttonDateFilterClear.link--accent.small((keydown.enter)="clearAllFilterGroup('date')", [tabindex]="hasAppliedFilters('date') ? 2 : -1", [ngClass]="{ 'disable': !hasAppliedFilters('date') }", (click)="clearAllFilterGroup('date')", aria-label="Clear, click to clear applied Date filter") Clear
+        a#buttonDateFilterClear.link--accent.small(*ngIf="hasAppliedFilters('date')", (keydown.enter)="clearAllFilterGroup('date')", [tabindex]="hasAppliedFilters('date') ? 2 : -1", (click)="clearAllFilterGroup('date')", aria-label="Clear, click to clear applied Date filter") Clear
```

Comment:
```suggestion
        a#buttonDateFilterClear.link--accent.small(*ngIf="hasAppliedFilters('date')", (keydown.enter)="clearAllFilterGroup('date')", [tabindex]="hasAppliedFilters('date') ? 2 : -1", (click)="clearAllFilterGroup('date')", aria-label="Clear, click to clear applied Date filter.") Clear
```

Reply:
Changes made in the next commit `4d90ebb`.